### ISSUE TITLE
Inference using a LabelEncoder with input type int64 

### ISSUE THREAD

**Issue by niklas-mohr-jdas** [microsoft/onnxruntime#2897]
**Describe the bug**

I try to load an onnx model exported from scikit-learn model with a LabelEncoder in an onnxruntime session. This works when I use string tensors as input to the LabelEncoder. However, I get the error ```onnxruntime.capi.onnxruntime_pybind11_state.NotImplemented: [ONNXRuntimeError] : 9 : NOT_IMPLEMENTED : Could not find an implementation for the node LabelEncoder:LabelEncoder(2)```
when the input is a int64 tensor.

**Urgency**
Currently this is a blocker to use onnxruntime for our productive model export that we would like to start using in Q1 2020.

**System information**
- ONNX Runtime version: 1.1.0
- onnx: 1.6.0
- skl2onnx: 1.6.0
- numpy: 1.17.2
- scikit-learn: 0.22.1
- OS: Debian 9
- Python 3.6

**To Reproduce**
This following code snippet reproduces the mentioned issue

```
import pytest
import numpy as np
import onnx
import onnxruntime
from sklearn.preprocessing import LabelEncoder
from skl2onnx import convert_sklearn
from skl2onnx.common.data_types import Int64TensorType, StringTensorType


@pytest.mark.parametrize(
    "input_values, tensor_type",
    [
        (np.array(["5", "2", "3"]), StringTensorType([None, 1])),
        (np.array([5, 2, 3]), Int64TensorType([None, 1])),
    ],
)
def test_onnx_label_encoder(input_values, tensor_type):
    le = LabelEncoder()
    le.fit(input_values)
    initial_types = [("TensorInput", tensor_type)]
    model_onnx = convert_sklearn(le, initial_types=initial_types, target_opset=11)
    onnx.checker.check_model(model_onnx)
    sess = onnxruntime.InferenceSession(model_onnx.SerializeToString())
```

Full output of the above test is
```
tests/onnxruntim_issue.py::test_onnx_label_encoder[input_values0-tensor_type0] PASSED                                                                    [ 50%]
tests/onnxruntim_issue.py::test_onnx_label_encoder[input_values1-tensor_type1] FAILED                                                                    [100%]

=========================================================================== FAILURES ===========================================================================
_____________________________________________________ test_onnx_label_encoder[input_values1-tensor_type1] ______________________________________________________
tests/onnxruntim_issue.py:23: in test_onnx_label_encoder
    sess = onnxruntime.InferenceSession(model_onnx.SerializeToString())
/home/docker/venv3.6/lib/python3.6/site-packages/onnxruntime/capi/session.py:25: in __init__
    self._load_model(providers)
/home/docker/venv3.6/lib/python3.6/site-packages/onnxruntime/capi/session.py:43: in _load_model
    self._sess.load_model(providers)
E   onnxruntime.capi.onnxruntime_pybind11_state.NotImplemented: [ONNXRuntimeError] : 9 : NOT_IMPLEMENTED : Could not find an implementation for the node LabelEncoder:LabelEncoder(2)
```

**Expected behavior**

The test with input type int64 should pass.

**Comment by skottmckay** [comment]
LabelEncoder supports int64 as input. 
There's no support implemented for using the same type as both the key and value type, which is the issue with this model when the input is int64. 

Is there a real use case where that functionality is required?

**Comment by niklas-mohr-jdas** [comment]
@skottmckay yes, I specifically need int64 to int64 support: the use case is a set of non ordered integers (e.g. 101, 50235, 100348) which we map to (0, 1, 2) using the LabelEncoder to feed them into an embedding layer.

**Comment by niklas-mohr-jdas** [comment]
Would it be ok if I try to contribute the changes the support this?

**Comment by hariharans29** [comment]
Yes, please feel free to contribute the change needed to support it. Thanks.

**Comment by hariharans29** [comment]
Closed via #2926
